recipe/meta.yaml

```
@@ -17,14 +17,20 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
-    - make
+    - make  # [not win]
     - ninja  # [win]
+  host:
+    # ensmallen is a C++ header only library. We don't technically need these in hosts, but they
+    # allow ot inject run_exports.
+    - intel-openmp  # [win]
     - llvm-openmp  # [osx]
     - libgomp      # [linux]
-  host:
     - armadillo >=9.800.0
-    - libblas
   run:
+    # Pins through run_exports of packages in host.
+    - intel-openmp  # [win]
```

```suggestion
    - {{ pin_compatible('intel-openmp') }} # [blas_impl == "mkl"]
```
intel-openmp doens't have run_exports.

---

Thanks. In this case, we can just remove intel-openmp from host and do `intel-openmp {{ mkl }} # [blas_impl == "mkl"]` here.